README.Rmd

```diff
@@ -36,6 +36,8 @@ We encourage contributions on any part of the book, including:
 
 Please see [style.md](https://github.com/Robinlovelace/geocompr/blob/master/style.md) for the book's style.
 
+If you would like to add to the references, please join Zotero and add your citation to the open [geocompr group library](https://www.zotero.org/groups/418217/energy-and-transport/items/collectionKey/9K6FRP6N).
```

Comment:
@Robinlovelace I've never used Zotero before (I mostly use Mendeley) - but I'm definitely going to check this approach.